pkg/metrics/metrics.go

```diff
@@ -0,0 +1,63 @@
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+const (
+	separator = "_"
+
+	subsystem = "stonesoup_sprayproxy"
```

Comment:
I'd prefer to keep this agnostic to StoneSoup. This proxy is simple enough that anyone can use it.

```suggestion
	subsystem = "sprayproxy"
```

Reply:
part of next push (but deferring from your commit suggestion)